src/DIRAC/Core/DISET/private/Service.py

```diff
@@ -64,7 +64,7 @@ def __init__(self, serviceData):
         self.__maxFD = 0
         self.activityMonitoring = False
         # Check if monitoring is enabled
-        if "Monitoring" in Operations().getMonitoringBackends(monitoringType="Service"):
+        if "Monitoring" in Operations().getMonitoringBackends(monitoringType="ServiceMonitoring"):
```

Comment:
that should go in the wiki if the option name changes

Reply:
Already done